CONTRIBUTING.md

```diff
@@ -85,12 +85,18 @@ environments. Our recommended setup for contributing is:
   ``cd metpy``
 * Connect your repository to the upstream (main project).
   ``git remote add unidata https://github.com/unidata/metpy.git``
-* Create the development environment by running
-  ``conda env create -n devel --file ci/requirements.txt --file ci/extra_requirements.txt
-  --file ci/test_requirements.txt``.
-  This will install all the necessary packages for development.
-* Activate our new development environment by running ``conda activate devel``
-* Make an editable install of MetPy by running ``pip install -e .``
+* Create a new conda environment for us to configure, and give it a name with
```

Comment:
Would be good to standardize either ending the bullets with "." or not. Right now it's split 2/3.